src/styles/SectionHero.less

```diff
@@ -0,0 +1,23 @@
+.hero {
+    color: #333;
+    border-bottom: solid 1px black;
+    
+    &--description {
+        font-size: 2.5rem;
+
+        &--colored {
+            color: #8b51c6;
+        }
+    }
+
```

Comment:
Use color variables in separate file?

Reply:
Legger til color.less nå slik det er gjort i repoet til rodekors.